newIDE/app/scripts/import-zipped-file.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This script will extract a zipped file of a prebuilt external extension to GD that we don't want added to git history
```

Comment:
As you mentioned "external extension", your file name should reflect this by being named "import-zipped-electron-extension" or something like this - to state that it is *not* general purpose.